### DISCUSSION TITLE
Multiple workspaces with separate configuration

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#3250] [Ideas]
**security concern in accessing documents in the vector database **
At present time, the vector database is configured once in the environment configuration file.you can have only one vector database at a time

**Describe the solution you'd like**
We should be able to define multiple workspaces with their own models, documents, 
Prompts and vector database.

**Describe alternatives you've considered**
Install multiple instances of open WebUI but overcharge the CPU and ram and the update procedure.

**Comment by justinh-rahb** [comment]
Teams is coming eventually, hopefully in 0.4.0

**Reply by BartEvers** [reply]
this would be an extremely useful feature, as there is currently no way to build multiple data collections. could you please take this in the next release? when would this feature be available? thank you!

**Comment by tjbck** [comment]
like @justinh-rahb seems relevant to our planned Teams feature + you should be able to achieve something similar with pipelines once #2972 is implemented.

**Comment by flefevre** [comment]
Thanks.

How can I have information on planned features to avoid to create tickets when you have already think the feature?

For instance the'team' feature?

If I do understand,it is preferable to create discussion at first, rather tickets for feature directly.
Do you confirm the process?


Thanks

**Comment by flefevre** [comment]
Will team feature also include the notion of 'manager' of a given workspace and so able to manage
Prompt, document and or models assistants?

**Comment by flefevre** [comment]
Will a team able to share their discussion?
So you can interact between different human users and ai assistantS in the same discussion lane.

**Comment by williamzebrowskI** [comment]
Would absolutely love a Team feature.  I love OpenWeb UI but I'm currently looking at other tools that can serve up multiple workspaces (or Teams) because of my use case.

**Reply by flefevre** [reply]
I am in the same configuration, without team workspace I do not know how to deploy open web UI for the different laboratories since the scientist will have everything together.
I have created a ticket here, perhaps you could add your 2-cents?

https://github.com/open-webui/open-webui/discussions/7082
